corehq/apps/hqcase/export.py

```diff
@@ -56,7 +60,7 @@ def get_matching_owner(case):
             return get_owner_id(case)
 
     def might_be_relevant(case):
-        return not filter_group or get_owner_id(case) in owner_ids
+        return get_owner_id(case) in owner_ids
```

Comment:
the use of the `owner_ids` in this closure is pretty tricky - might suggest removing the function entirely and just replacing the if condition with the contents of the function